### ISSUE TITLE
feed output not WCAG 2.0 compliant

### ISSUE THREAD

**Issue by kphunter** [cogdog/feed2js#9]
Hi,

According to http://www.w3.org/TR/WCAG20-TECHS/SCR21.html, the document.write() method does not work with XHTML when served with the correct MIME type (application/xhtml+xml). I'm looking into forking the code with DOM functions but might need a little help...

Thanks,

Kyle
